src/viam/app/viam_client.py

```diff
@@ -24,14 +25,46 @@ class ViamClient:
         ViamClient.create_from_dial_options(...)
     """
 
+    @classmethod
+    async def create_from_env_vars(cls, dial_options: Optional[DialOptions] = None, app_url: Optional[str] = None) -> Self:
+        """Create `ViamClient` using credentials set by a module.
```

Comment:
I think instead of "set by a module" maybe say "using credentials set in the environment as `VIAM_API_KEY` and `VIAM_API_KEY_ID`"